profiles/hemantwasthere.json

```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://github.com/hemantwasthere.png",
```

Comment:
Here you should provide an image of your profile and not your profile photo itself.

What do you think of this image?

![image](https://user-images.githubusercontent.com/51391473/191279633-3cb321f7-af64-4743-bc6a-a590a5081afa.png)

```suggestion
  "image": "https://user-images.githubusercontent.com/51391473/191279633-3cb321f7-af64-4743-bc6a-a590a5081afa.png",
```